app/models/publications/itt_monthly_report_generator.rb

```diff
@@ -0,0 +1,81 @@
+module Publications
+  class ITTMonthlyReportGenerator
+    attr_reader :generation_date, :first_cycle_week, :report_expected_time, :cycle_week
+
+    def initialize(generation_date: Time.zone.now)
+      @generation_date = generation_date
+      @first_cycle_week = CycleTimetable.find_opens.beginning_of_week
+      @report_expected_time = @generation_date.beginning_of_week(:sunday)
+      @cycle_week = (@report_expected_time - first_cycle_week).seconds.in_weeks.round
+    end
+
+    def to_h
+      {
+        meta:,
+        candidate_headline_statistics: {
+          title: I18n.t('publications.itt_monthly_report_generator.candidate_headline_statistics.title'),
+          data: candidate_headline_statistics,
+        },
+      }
+    end
+
+    def meta
+      {
+        generation_date:,
+        period:,
+        cycle_week:,
+      }
+    end
+
+    def period
+      "From #{first_cycle_week.to_fs(:govuk_date)} to #{report_expected_time.to_fs(:govuk_date)}"
+    end
+
+    def candidate_headline_statistics
+      application_metrics = DfE::Bigquery::ApplicationMetrics.candidate_headline_statistics(cycle_week:)
+
+      {
+        submitted: {
```

Comment:
I chosen to hardcoded the status and title instead of going with a loop and magic finding translation. I found this easier to read for anyone in the team without too much magic. But let me know if you have a strong opinion to dynamic status and get their counterpart on application metrics. Is possible but too magic 🪄 🪄 🪄 🪄

Reply:
Nevermind what I wrote above. For all the other sections I need some magic to determine the columns. So for this PR will he hardcoded like these but for the next sections I'd need a magic to determine grouped statuses with the application metrics columns from bigquery. But I will try to make simple as possible.